### PR TITLE
:sparkles: Add organizer with popup in form - ref #3587

### PR DESCRIPTION
**Pull request by TheoLechemia**
## Description

Add a popup button to add new organizer

## Related Issue

#3587

## Checklist

- [x] I have followed the guidelines in our [Contributing document](https://geotrek.readthedocs.io/en/latest/CONTRIBUTING.html)
- [ ] My code respects the Definition of done available in the [Development section of the documentation](https://geotrek.readthedocs.io/en/latest/contribute/development.html#definition-of-done-for-new-model-fields)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes
- [x] I added an entry in the changelog file
- [x] My commits are all using prefix convention (emoji + tag name) and references associated issues
- [x] I added a label to the PR corresponding to the perimeter of my contribution
- [x] The title of my PR mentionned the issue associated

### CODE REVIEW

File: geotrek/tourism/models.py [hunk 1]
```diff
@@ -387,7 +388,7 @@ def __str__(self):
 
 
 class TouristicEventOrganizer(TimeStampedModelMixin):
-    label = models.CharField(verbose_name=_("Label"), max_length=256)
+    label = models.CharField(verbose_name=_("OrganizerLabel"), max_length=256)
```
Comment (submarcos) [inline geotrek/tourism/models.py:391]: if you change a tanslation, you should update traslation files and set french translation at least.

Byt maybe it's a mistake

File: geotrek/tourism/views.py [hunk 2]
```diff
@@ -275,6 +279,23 @@ def get_context_data(self, **kwargs):
...
 
+class TouristicEventOrganizerCreatePopup(CreateView):
+    model = TouristicEventOrganizer
+    form_class = TouristicEventOrganizerFormPopup
+
+    @method_decorator(login_required)
+    def dispatch(self, request, *args, **kwargs):
+        if not request.user.has_perm("tourism.add_touristiceventorganizer"):
+            raise PermissionDenied
+        return super().dispatch(request, *args, **kwargs)
```
[the remark this answers is not in the document]
Comment (submarcos) [reply geotrek/tourism/views.py:290]: non si codecov dit que c'est pas couvert c'es tque çà ne l'est pas. il y a deux cas a tester, un avec l'utilisateur qui a la permission, l'autre avec un utilisateur qui ne l'a pas

[the remark this answers is not in the document]
Comment (submarcos) [reply geotrek/tourism/views.py:290]: dans ton test tu ne testes pas avec un utilisateur qui n'a pas la permission mais avec un utilisateur anonyme, qui n'arrive meme pas jusqu'a ucode car il est ejecté par le décorateur login_required

File: geotrek/tourism/models.py (hunk 1 above)
Comment (TheoLechemia) [reply geotrek/tourism/models.py:391]: Ok.
Nop not a mistake. The `Label` has already a translation which was not appropriate for this context

**Comment by submarcos** [comment]
Derniere remarque, garde Label comme verbose name du modele (et dans la migration, et remake messages)
car c'est comme çà qu'on nomme / traduit tous les labels dans l'app (coherence)

File: geotrek/tourism/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -459,5 +459,10 @@ msgstr ""
 msgid "Date"
 msgstr "Datum"
 
+#, fuzzy
```
Comment (submarcos) [inline geotrek/tourism/locale/de/LC_MESSAGES/django.po:462]:
```suggestion
```

File: geotrek/tourism/locale/de/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -459,5 +459,10 @@ msgstr ""
...
+#| msgid "Organizer"
```
Comment (submarcos) [inline geotrek/tourism/locale/de/LC_MESSAGES/django.po:463]:
```suggestion
```

File: geotrek/tourism/locale/de/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -459,5 +459,10 @@ msgstr ""
...
+msgid "Add Organizer"
+msgstr "Organisator"
```
Comment (submarcos) [inline geotrek/tourism/locale/de/LC_MESSAGES/django.po:465]:
```suggestion
msgstr ""
```

File: geotrek/tourism/locale/fr/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -464,5 +464,10 @@ msgstr "Total"
 msgid "Date"
 msgstr "Date"
 
+#, fuzzy
```
Comment (submarcos) [inline geotrek/tourism/locale/fr/LC_MESSAGES/django.po:467]:
```suggestion
```

File: geotrek/tourism/locale/fr/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -464,5 +464,10 @@ msgstr "Total"
...
+#| msgid "Organizer"
```
Comment (submarcos) [inline geotrek/tourism/locale/fr/LC_MESSAGES/django.po:468]:
```suggestion
```

File: geotrek/tourism/locale/fr/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -464,5 +464,10 @@ msgstr "Total"
...
+msgid "Add Organizer"
+msgstr "Organisateur"
```
Comment (submarcos) [inline geotrek/tourism/locale/fr/LC_MESSAGES/django.po:470]:
```suggestion
msgstr "Ajouter un organisateur"
```

File: geotrek/tourism/locale/it/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -458,5 +458,10 @@ msgstr ""
 msgid "Date"
 msgstr "Data"
 
+#, fuzzy
```
Comment (submarcos) [inline geotrek/tourism/locale/it/LC_MESSAGES/django.po:461]:
```suggestion
```

File: geotrek/tourism/locale/it/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -458,5 +458,10 @@ msgstr ""
...
+#| msgid "Organizer"
```
Comment (submarcos) [inline geotrek/tourism/locale/it/LC_MESSAGES/django.po:462]:
```suggestion
```

File: geotrek/tourism/locale/it/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -458,5 +458,10 @@ msgstr ""
...
+msgid "Add Organizer"
+msgstr "Organizzatore"
```
Comment (submarcos) [inline geotrek/tourism/locale/it/LC_MESSAGES/django.po:464]:
```suggestion
msgstr ""
```

File: geotrek/tourism/tests/test_forms.py [hunk 12]
```diff
@@ -1,5 +1,10 @@
 
+
```
Comment (submarcos) [inline geotrek/tourism/tests/test_forms.py:2]:
```suggestion
```

File: geotrek/tourism/urls.py [hunk 13]
```diff
@@ -22,6 +22,8 @@
     path('api/<lang:lang>/touristiccategories.json', tourism_views.TouristicCategoryView.as_view(), name="touristic_categories_json"),
     path('api/<lang:lang>/touristiccontents/<int:pk>/meta.html', tourism_views.TouristicContentMeta.as_view(), name="touristiccontent_meta"),
     path('api/<lang:lang>/touristicevents/<int:pk>/meta.html', tourism_views.TouristicEventMeta.as_view(), name="touristicevent_meta"),
+    path('popup/add/organizer/', tourism_views.TouristicEventOrganizerCreatePopup.as_view(), name='organizer_add'),
+
```
Comment (submarcos) [inline geotrek/tourism/urls.py:26]:
```suggestion
```

**Comment by submarcos** [comment]
replaced by #3962 with rebase and ready to merge

**Pull request merged**